logic/labrynna.go

```diff
@@ -158,13 +163,16 @@ var labrynnaNodes = map[string]*Node{
 	"ridge base past east": Or("target carts",
 		And("lynna city", Or("feather", "ages"), "mermaid suit"),
 		And("ridge mid past", "feather", "brother emblem"),
-		And("ridge mid present", "ages"),
+		"rolling ridge east tree",
+		And("ridge base present", "ages"),
```

Comment:
RBPE is reachable from "ridge mid present" because a ledge between "ridge mid past" and "ridge base past east" was added to prevent softlocking in that area.